src/test/resources/harness-config.yml

```diff
@@ -3,55 +3,80 @@ context: testContext
 
 databasesUnderTest:
   - name: postgresql
-    version: 9
-    url: jdbc:postgresql://localhost:5433/lbcat
+    version: 13
+    url: jdbc:postgresql://localhost:5437/lbcat
     username: lbuser
     password: LiquibasePass1
 
   - name: postgresql
     version: 12
-    url: jdbc:postgresql://localhost:5434/lbcat
+    url: jdbc:postgresql://localhost:5436/lbcat
     username: lbuser
     password: LiquibasePass1
 
-  - name: mysql
-    version: 5.7
-    url: jdbc:mysql://localhost:33062/lbcat
+  - name: postgresql
+    version: 11
+    url: jdbc:postgresql://localhost:5435/lbcat
     username: lbuser
     password: LiquibasePass1
 
-  - name: mysql
-    version: 8
-    url: jdbc:mysql://localhost:33061/lbcat
+  - name: postgresql
+    version: 10
+    url: jdbc:postgresql://localhost:5434/lbcat
     username: lbuser
     password: LiquibasePass1
 
-  - name: mariadb
-    version: 10.3
-    url: jdbc:mariadb://localhost:33063/lbcat
+  - name: postgresql
+    version: 9
+    url: jdbc:postgresql://localhost:5433/lbcat
     username: lbuser
     password: LiquibasePass1
 
-  - name: oracle
-    version: 18.4.0
-    url: 'jdbc:oracle:thin:@localhost:1521/xe'
-    username: C##LIQUIBASE
-    password: Secret321
-
-  - name: cockroachdb
-    version: 20.2
-    url: jdbc:postgresql://localhost:26257/lbcat
+  - name: postgresql
+    version: 9.5
+    url: jdbc:postgresql://localhost:5432/lbcat
     username: lbuser
-    password:
+    password: LiquibasePass1
 
-  - name: cockroachdb
-    version: 20.1
-    url: jdbc:postgresql://localhost:26258/lbcat
-    username: lbuser
-    password:
 
-  - name: mssql
-    version: 2017
-    url: jdbc:sqlserver://localhost:1433;databaseName=lbcat
-    username: lbuser
-    password: LiquibasePass1
+#  - name: mysql
+#    version: 5.7
+#    url: jdbc:mysql://localhost:33062/lbcat
+#    username: lbuser
+#    password: LiquibasePass1
+#
```

Comment:
Same request in this file as well @r2liquibase -- please add back (uncomment) all the other dbs that you have commented out. Otherwise these tests will not be executed by default.